aws-cpp-sdk-core/source/utils/crypto/openssl/CryptoImpl.cpp

```diff
@@ -35,16 +35,19 @@ namespace Aws
         {
             namespace OpenSSL
             {
+#if OPENSSL_VERSION_NUMBER < 0x10100003L
```

Comment:
I believe the version numbers should have a 1 (or a zero, doesn't really matter) as the last digit, so that people on 1.1b and 1.1a work as well.